master/internal/api_trials.go

```diff
@@ -496,29 +497,110 @@ func (a *apiServer) GetTrial(_ context.Context, req *apiv1.GetTrialRequest) (
 		return nil, errors.Wrapf(err, "failed to get trial %d", req.TrialId)
 	}
 
-	workloads := apiv1.GetTrialWorkloadsResponse{}
-	switch err := a.m.db.QueryProtof(
-		"proto_get_trial_workloads",
-		[]interface{}{
-			db.OrderByToSQL(apiv1.OrderBy_ORDER_BY_ASC),
-			db.OrderByToSQL(apiv1.OrderBy_ORDER_BY_ASC),
-		},
-		&workloads,
-		req.TrialId,
-		nil,
-		nil,
-	); {
+	return resp, nil
+}
+
+func (a *apiServer) appendToMetrics(metrics []*apiv1.SummarizedMetric, m *apiv1.SummarizedMetric,
+	metricSeries []lttb.Point) []*apiv1.SummarizedMetric {
+	for _, in := range metricSeries {
+		out := apiv1.DataPoint{
+			Batches: int32(in.X),
+			Value:   in.Y,
+		}
+		m.Data = append(m.Data, &out)
+	}
+	if len(m.Data) > 0 {
+		return append(metrics, m)
+	}
+	return metrics
+}
+
+func (a *apiServer) MultiTrialSample(trialID int32, metricNames []string,
+	metricType apiv1.MetricType, maxDatapoints int, startBatches int,
+	endBatches int, log_scale bool) ([]*apiv1.SummarizedMetric, error) {
+	var metricSeries []lttb.Point
+	var startTime time.Time
+	// var endTime time.Time
+	var err error
+
+	var metrics []*apiv1.SummarizedMetric
+
+	for _, name := range metricNames {
+		if metricType != apiv1.MetricType_METRIC_TYPE_VALIDATION {
```

Comment:
why not check if == training?

Reply:
if the metricType is Training or Unspecified (all types), then we should return training metrics.  I agree this is confusing so I will change it to an OR